mdata/cache/if.go

```diff
@@ -7,16 +7,23 @@ import (
 )
 
 type Cache interface {
```

Comment:
could you update this type and add sensible variable names for all parameters. (and also for the methods of the implementations) thanks.

Reply:
I'm not sure what you mean with `and also for the methods of the implementations`. Are the current ones not sensible enough?
Anyway, I've added the names to the interfaces: https://github.com/grafana/metrictank/pull/555/commits/2b1ad9bd8c14f60b5d27e1aa16d893ebfdce4950

Reply:
i added a few f545bc416b14dc6a36d16c3cc05a8f234b30d126 but yeah most implementations were already good